src/main/java/seedu/address/logic/parser/schedule/CliSyntax.java

```diff
@@ -9,5 +9,6 @@
     /* Prefix definitions */
     public static final Prefix PREFIX_CLIENT_INDEX = new Prefix("c/");
     public static final Prefix PREFIX_SESSION_INDEX = new Prefix("s/");
+    public static final Prefix PREFIX_IS_PAID = new Prefix("p/");
```

Comment:
not sure if a different prefix should be used instead because there are a number of commands with prefix p/ already, just with starting different command word

Reply:
yep, it represents phone in add client cmd. Do you have any suggestion for this?
im thinking of paid/ or pay/, but afraid that those are too long

Reply:
yeap and period is sview as well, hmm maybe pd/ like payment done? but its solely just to differentiate with the other prefixes so that it won't be too confusing